src/Paket.Core/Dependencies/RemoteDownload.fs

```diff
@@ -266,11 +266,13 @@ let downloadRemoteFiles(remoteFile:ResolvedSourceFile,destination) = async {
     | Origin.HttpLink(origin), _ ->
         let url = origin + remoteFile.Commit
         let authentication = auth remoteFile.AuthKey url
+        let timeout = (Some(TimeSpan.FromDays(1.0)))
```

Comment:
o_O

Reply:
It is kind of `Infinity`, sometimes internet connection is really pure... =) I can spend `20min` just to download `8Mb` of new `paket.exe` from Github. Do you have better default for `infinity` timeout?